contracts/accounts/Paymaster.sol

```diff
@@ -137,8 +137,8 @@ contract Paymaster is
      */
     function validatePaymasterUserOp(
         UserOperation calldata userOp,
-        bytes32 userOpHash,
-        uint256 maxCost
+        bytes32, // _userOpHash, // intentionally not used
+        uint256 // _maxCost // intentionally not used
```

Comment:
don't do this, it messes up compilation and export of contract interfaces to other languages